### DISCUSSION TITLE
Error dotnet ef database update

### DISCUSSION THREAD

**Discussion by luistring8** [balta-io/2811#39] [Q&A] [answered]
Olá, não estou conseguindo executar o comando

<img width="1552" alt="Captura de Tela 2022-09-04 às 3 59 28 PM" src="https://user-images.githubusercontent.com/68756072/188331403-6e46c8fc-9fb5-4b07-90c6-3247bea833e8.png">

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @luistring8 qual comando?

**Comment by luistring8** [comment]
Bom dia, professor?

dotnet ef database update

sabe me dizer se esse erro pode ter relaçao ao processador M1, pq aparentemente está tudo ok no docker...

**Reply by andrebaltieri** [reply]
Aqui tem os detalhes e correção deste erro (Última parte)
👉 https://balta.io/blog/sql-server-docker

**Reply by luistring8** [reply]
Deu Certo! Obrigado..
